### PR TITLE
[SRM-702] Fix for revision loading throws error on illegal offset type.

### PR DESCRIPTION
**Pull request by MdNadimHossain**
### JIRA
https://digital-vic.atlassian.net/browse/SRM-702

### Issue
Some of the $vid values returned as object and which causes illegal offset type error when calling the `loadRevision` function.
Scenario - Go to a node in any CMS and click on the revision tab. Then click on any previous revision and then the CMS will throw error.

### Changes
9.4 returns the vid directly from here - `$revision = $this->routeMatch->getParameter('node_revision');`
https://github.com/dpc-sdp/tide_site/issues/121

### CODE REVIEW

**Review by mayurngondhkar — APPROVED** [review]
LGTM

File: modules/tide_site_preview/src/Plugin/Block/PreviewLinksBlock.php [hunk 1]
```diff
@@ -234,8 +234,7 @@ protected function getCurrentNode() : ?NodeInterface {
     if ($route_name === 'entity.node.revision') {
       try {
         $vid = $this->routeMatch->getParameter('node_revision');
```
Comment (govindmaloo) [inline modules/tide_site_preview/src/Plugin/Block/PreviewLinksBlock.php:236]: This will directly return the node revision entity post Drupal 9.3.x

Comment (MdNadimHossain) [reply modules/tide_site_preview/src/Plugin/Block/PreviewLinksBlock.php:236]: yes, I think that is why it used to work, but after updating to 9.4.x it started returning object as well. I have seen similar issues raised against d10 as well. Hopefully that will at least fix the issue from our side. @govindmaloo

**Comment by MdNadimHossain** [comment]
Hey @govindmaloo I have updated the code, please have a look

**Review by vincent-gao — APPROVED** [review]
LGTM. 
We may replace the node logic by using `context` further.

**Pull request merged**